packages/fx-core/package.json

```diff
@@ -211,7 +214,8 @@
     "ts-node": "^10.9.2",
     "tslib": "^2.3.1",
     "typescript": "4.7.4",
-    "webpack": "^5.61.0"
```

Comment:
Is webpack still used in fx-core?

Reply:
Maybe not used, I will check and remove it in another PR.